src/GitHub.VisualStudio/UI/Views/PullRequestCreationView.xaml.cs

```diff
@@ -20,6 +21,8 @@ public PullRequestCreationView()
 
             this.WhenActivated(d =>
             {
+                d(this.BindCommand(ViewModel, vm => vm.Cancel, v => v.cancelButton));
```

Comment:
Yeah, I'm trying to clear away the rxui bindings and use xaml. They were needed before because the view model was being set incorrectly, but not anymore. With xaml bindings, don can set up all the data himself when he mocks things up and we don't have to do extra steps to hook it up. :smile: